deps/rabbit_common/src/rabbit_misc.erl

```diff
@@ -1399,6 +1402,25 @@ rpc_call(Node, Mod, Fun, Args, Timeout) ->
 get_gc_info(Pid) ->
     rabbit_runtime:get_gc_info(Pid).
 
+-spec raw_read_file(Filename) -> {ok, Binary} | {error, Reason} when
```

Comment:
Note that while we do have `rabbit_file`, it is a server module and common does not depend on server.